### PR TITLE
Test different versions of Symfony

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Related to #49.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ matrix:
       env: PHP_CS_FIXER_VERSION=1.*
     - php: 5.6
       env: PHP_CS_FIXER_VERSION=2.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev@dev as 2.8"
```
Comment (GrahamCampbell) [inline .travis.yml]: hax, lol

**Comment by GrahamCampbell** [comment]
> StyleCI fixers are out of date. Run styleci:config:update command to fix it.

Sorry, lol.

File: .travis.yml (hunk 1 above)
Comment (Soullivaneuh) [inline .travis.yml]: Shhht! :-)

**Comment by Soullivaneuh** [comment]
> > StyleCI fixers are out of date. Run styleci:config:update command to fix it.

> Sorry, lol.

I can't bear people who work. :-P

BTW, I'm working on a separate repository for that in order to avoid to put lot of patch release and have painful version constraint with that.

**Pull request merged**